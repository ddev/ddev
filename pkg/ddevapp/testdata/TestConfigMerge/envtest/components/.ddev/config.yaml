# Simplest
name: TestConfigMerget1

web_environment:
- ORIG1=orig1
- ORIG_TO_BE_OVERWRITTEN=orig
