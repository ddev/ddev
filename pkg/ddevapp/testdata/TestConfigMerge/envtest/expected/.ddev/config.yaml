name: TestConfigMerget1
web_environment:
  - LAST=last_1
  - LAST2=last_2
  - ORIG1=orig1
  - ORIG_TO_BE_OVERWRITTEN=last
